element_deeplabcut/model.py

```diff
@@ -508,6 +514,11 @@ def infer_output_dir(cls, key, relative=False, mkdir=False):
             str(v)
             for v in (_linking_module.Device & recording_key).fetch1("KEY").values()
         )
+        if get_dlc_processed_data_dir():
+            processed_dir = Path(get_dlc_processed_data_dir())
+        else:  # if processed not provided, default to where video is
+            processed_dir = root_dir

```

Comment:
This if-else would be redundent if get_dlc_processed_data_dir defaulted to get_dlc_root_data_dir when it's not defined.